### DISCUSSION TITLE
part-ii/2019-102/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#306] [Question comments]
Part II 2019 Number Theory: Paper 1, Section I, I 

https://questions.tripos.org/part-ii/2019-102/

**Comment by callumhobbis** [comment]
<details>
<summary>Sketch solution</summary>

Almost all bookwork. For the last bit, it's not too bad. We just use the definition of 35 being a Fermat pseudoprime to base $b$, then apply the Chinese remainder theorem and use Fermat's little theorem to get $b^2\equiv 1\enspace(\text{mod }5)$ and $b^4\equiv1\enspace(\text{mod }7)$. This gives $b\equiv-1,1\enspace(\text{mod }5)$ and $b^2\equiv-1,1\enspace(\text{mod }7)$. Since $(\tfrac{-1}7)=-1$ (or by brute force), we get $b^2\equiv-1,1\enspace(\text{mod }7)$. So we get $b\equiv1,6,29,34\enspace(\text{mod }35)$.

</details>

**Reply by walrus1729** [reply]
<details>
<summary>Small comment</summary>

All correct. One can also notice that if $b^4 \equiv 1 \pmod{7}$ (which we have found), and $b^6 \equiv 1 \pmod{7}$ (by Fermat's little theorem), then this forces $b^6 b^{-4} \equiv b^2 \equiv 1 \pmod{7}$ directly, so there is no need to compute $(\frac{-1}{7})$.

</details>
